.github/workflows/js_tests.yml

```
@@ -0,0 +1,35 @@
+name: JavaScript
+on:
+  pull_request:
+    paths:
+      - 'webpack/**'
+      - 'package.json'
+      - '.github/workflows/js_tests.yml'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [
+          10,
+          12,
+        ]
```

This is the YAML syntax for it, exactly the same as in `.travis.yml`.
```suggestion
        node-version:
          - 10
          - 12
```

---

makes sense :)